en/frames_rover/cyber_rover.md

```diff
@@ -0,0 +1,55 @@
+# Cyber Rover (ARK Electronics)
+
+The "Cyber Rover" as we've come to call it is built on the frame of a Traxxas TRX-4 Bronco. We decided to use this as the base of our rover largely because of the large chassis for mounting multiple components, and the high travel adjustable suspension. We wanted to be able to use this mostly anywhere without fear of it getting stuck or slowed down.
+
+![Traxxas_TRX4_1](../../assets/airframes/rover/cyber_rover/cyber_rover_1.jpeg)
+![Bronco](../../assets/airframes/rover/cyber_rover/traxxas_trx4_1.jpeg)
+
+## Parts List
+
+- [Traxxas TRX-4 Bronco](https://traxxas.com/products/models/electric/trx-4-bronco)
+- [ARK Flow](../dronecan/ark_flow.md)
```

Comment:
Where the files exist in our docs I have linked there. Usually there is more useful setup guide etc stuff there.